### ISSUE TITLE
coordinate frame convention

### ISSUE THREAD

**Issue by geyang** [nasa-jpl/m2020-urdf-models#3]
Hi @gkjohnson 

I am visualizing this model and it seems that convention  is z-down. Is this expected?

Thanks!

**Comment by gkjohnson** [comment]
Yes many JPL and NASA robotics use +Z down as convention. See the coordinate system conventions here for the MSL rover on slide 3 (same used for M2020):

https://pds-imaging.jpl.nasa.gov/documentation/MSL_COORDINATE_FRAMES.PDF

<img width="300" alt="Image" src="https://github.com/user-attachments/assets/a9ed8f8a-54e3-4220-8cb1-10cd3e453e56" />
